projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-js/pdf-js-wrapper.ts

```diff
@@ -57,7 +58,10 @@ export class PdfJsWrapper {
 
       this.pdfViewer.setDocument(pdfDocument);
       this.pdfViewer.linkService.setDocument(pdfDocument, null);
-
+      await pdfDocument.getMetadata()
```

Comment:
this is just an idea but maybe we can just set the title to the document id. that way images do not have to be set to null

Reply:
which id?

Reply:
the documents are normally passed with an id or the file name at the end of the url e.g.
document/123-1213123-123132-123 or assets/pdf-name.pdf

Reply:
I think since we cannot guarantee it will always be useful doesn't title make more sense? Happy to open the discussion up - I have discussed it with Lawrie and Linda to date

Reply:
if Linda and Lawrie are happy with it then we should go with that.